From 9a4938f38a1a5c6ab2f47ead56daeb97bf086495 Mon Sep 17 00:00:00 2001
From: Robert Eckelmann <longnoserob@gmail.com>
Date: Sun, 31 Mar 2024 14:22:17 +0900
Subject: [PATCH 107/142] arm64: dts: qcom: sdm845-xiaomi-beryllium: enable
 SLPI

Enable the SLPI dsp on the Xiaomi Pocophone F1 with Qualcom SDM845 SoC.

Signed-off-by: Robert Eckelmann <longnoserob@gmail.com>
---
 .../arm64/boot/dts/qcom/sdm845-xiaomi-beryllium-common.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/sdm845-xiaomi-beryllium-common.dtsi b/arch/arm64/boot/dts/qcom/sdm845-xiaomi-beryllium-common.dtsi
index 85e9efb579..e2ffd4b31b 100644
--- a/arch/arm64/boot/dts/qcom/sdm845-xiaomi-beryllium-common.dtsi
+++ b/arch/arm64/boot/dts/qcom/sdm845-xiaomi-beryllium-common.dtsi
@@ -757,3 +757,9 @@ &wifi {
 	qcom,snoc-host-cap-skip-quirk;
 	qcom,ath10k-calibration-variant = "xiaomi_beryllium";
 };
+
+
+&slpi_pas {
+	firmware-name = "qcom/sdm845/beryllium/slpi.mbn";
+        status = "okay";
+};
-- 
2.49.0

